lightning/src/ln/channel.rs

```diff
@@ -6103,8 +6126,20 @@ impl<'a, Signer: Sign, K: Deref> ReadableArgs<(&'a K, u32)> for Channel<Signer>
 			(17, announcement_sigs_state, option),
 			(19, latest_inbound_scid_alias, option),
 			(21, outbound_scid_alias, option),
+			(23, outbound_htlc_maximum_msat, option),
 		});
 
+		if outbound_htlc_maximum_msat.is_some() && counterparty_forwarding_info.is_some() {
+			let cf_info = counterparty_forwarding_info.as_ref().unwrap();
+			counterparty_forwarding_info = Some(CounterpartyForwardingInfo {
+				fee_base_msat: cf_info.fee_base_msat,
+				fee_proportional_millionths: cf_info.fee_proportional_millionths,
+				cltv_expiry_delta: cf_info.cltv_expiry_delta,
+				outbound_htlc_minimum_msat: cf_info.outbound_htlc_minimum_msat,
+				outbound_htlc_maximum_msat: outbound_htlc_maximum_msat
+			});
```

Comment:
Made this create a new `CounterpartyForwardingInfo` object with the retrieved `outbound_htlc_maximum_msat` value, to not require the `outbound_htlc_maximum_msat` field for `CounterpartyForwardingInfo` be mutable. But I can of course change that if we prefer the alternative!

Reply:
It should be less code to make it mutable - can just change the `as_ref()`, above, to `as_mut()` and it'll be mutable :)